Classes/Settings.cs

```diff
@@ -369,6 +330,36 @@ internal static int RssServerNumRecentEps
             }
         }
 
+        internal static string SetSaveFolder(Model.Programme progInfo)
```

Comment:
They are not properties anymore, because they take an argument. You yourself used Get and Set in those cases. I have thought about an overload of [], but should one think of SaveFolder as an array?